README.md

```diff
@@ -162,9 +181,9 @@ submodules, you will need to `git submodule update`, as above. If git
 throws an error like:
 
 ```
-fatal: No url found for submodule path 'resources/webidl2/test/widlproc' in .gitmodules
+fatal: No URL found for submodule path 'resources/webidl2/test/widlproc' in .gitmodules
 Failed to recurse into submodule path 'resources/webidl2'
-fatal: No url found for submodule path 'tools/html5lib' in .gitmodules
+fatal: No URL found for submodule path 'tools/html5lib' in .gitmodules
```

Comment:
Both this and the above are right (because they're copying git's error messages)